include/z64environment.h

```diff
@@ -111,27 +175,26 @@ typedef struct {
     /* 0xB4 */ u8 numLightSettings;
     /* 0xB8 */ EnvLightSettings* lightSettingsList; // list of light settings from the scene file
     /* 0xBC */ u8 blendIndoorLights; // when true, blend between indoor light settings when switching
-    /* 0xBD */ u8 unk_BD; // indoor light index
-    /* 0xBE */ u8 unk_BE; // prev indoor light index?
-    /* 0xBF */ u8 unk_BF;
+    /* 0xBD */ u8 lightSetting; // only used indoors or on override
+    /* 0xBE */ u8 prevLightSetting; // only used indoors or on override
+    /* 0xBF */ u8 lightSettingOverride;
     /* 0xC0 */ EnvLightSettings lightSettings;
-    /* 0xD6 */ u16 unk_D6;
-    /* 0xD8 */ f32 unk_D8; // indoor light blend weight?
-    /* 0xDC */ u8 unk_DC;
-    /* 0xDD */ u8 gloomySkyMode;
-    /* 0xDE */ u8 unk_DE; // gloomy sky state
-    /* 0xDF */ u8 lightningMode;
-    /* 0xE0 */ u8 unk_E0; // env sounds state
+    /* 0xD6 */ u16 lightBlendRateOverride;
+    /* 0xD8 */ f32 lightBlend;
+    /* 0xDC */ u8 lightBlendOverride;
+    /* 0xDD */ u8 stormRequest; // "rain_evt_trg"
+    /* 0xDE */ u8 stormState;
+    /* 0xDF */ u8 lightningState;
+    /* 0xE0 */ u8 timeSeqState;
     /* 0xE1 */ u8 fillScreen;
     /* 0xE2 */ u8 screenFillColor[4];
     /* 0xE6 */ u8 sandstormState;
     /* 0xE7 */ u8 sandstormPrimA;
     /* 0xE8 */ u8 sandstormEnvA;
     /* 0xE9 */ u8 customSkyboxFilter;
     /* 0xEA */ u8 skyboxFilterColor[4];
-    /* 0xEE */ u8 unk_EE[4];
-    /* 0xF2 */ u8 unk_F2[4];
-    /* 0xF6 */ char unk_F6[0x06];
+    /* 0xEE */ u8 precipitation[5];
```

Comment:
its probably ok? itll be really short compared to the rest, and someone reading this will probably be familiar with the enum_mzx construct to understand that its different than the rest